TodoApp/test/TodoApp.Services.Tests/Services/AddItemServiceTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Threading.Tasks;
+using NSubstitute;
+using NUnit.Framework;
+using TodoApp.Contracts.Factories;
+using TodoApp.Contracts.Models;
+using TodoApp.Contracts.Repositories;
+using TodoApp.Contracts.Services;
+using TodoApp.Services.Services;
+using TodoApp.Tests.Base.Comparers;
+
+namespace TodoApp.Services.Tests.Services
+{
+    [TestFixture]
+    internal class AddItemServiceTests
+    {
+        private IAddItemService _service;
+        private ITimeFactory _timeService;
+        private IGuidFactory _guidService;
+        private IItemRepository _repository;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _repository = Substitute.For<IItemRepository>();
+            _timeService = Substitute.For<ITimeFactory>();
+            _guidService = Substitute.For<IGuidFactory>();
+
+            _service = new AddItemService(_repository, _timeService, _guidService);
+        }
+
+        [Test]
+        public async Task AddItemAsync_NewItem_ReturnsSavedItem()
+        {
+            const string newItemText = "new item";
+            var currentTime = new DateTime(2018, 2, 5, 15, 0, 24);
```

Comment:
If you are using "magic numbers" and it doesn't make sense to name individual well-named variables (such as in this case), it usually helps to use parameter names - you easily prevent confusion (especially if you were working with Americans 😆) and make the code more self-explanatory (at cost of prolonged line of code).